Sources_v3/StreamChatUI/ChatChannelList/ChatChannelListViewController.swift

```diff
@@ -2,7 +2,116 @@
 // Copyright © 2020 Stream.io Inc. All rights reserved.
 //
 
-import Foundation
+import StreamChat
 import UIKit
 
-open class ChatChannelListViewController: UIViewController {}
+public typealias ChatChannelListViewController = _ChatChannelListViewController<DefaultExtraData>
```

Comment:
This naming is from hell 😱 `ChatChannelListController` vs `ChatChannelListViewController`

Reply:
haha, definitely 😄 You vote for `ChannelList`, do you?

Reply:
Yeah, let's not point fingers here 😅 😅 😅

Reply:
renamed in b41f9781372edb9336a4c1739e19ba8b267c615a

Reply:
@evsaev I think we should keep the `ViewController` or at least `VC` in the name 🤔

Reply:
renamed in 47b487dba9229c15cc2c2858b828f8e6cde80c65